rescript/plugin_setup.py

```diff
@@ -974,6 +980,39 @@
 )
 
 
+plugin.methods.register_function(
+    function=get_unite_data,
+    inputs={},
+    parameters={
+        'version': Str % Choices(['9.0', '8.3', '8.2']),
+        'taxon_group': Str % Choices(['fungi', 'eukaryotes']),
+        'cluster_id': Str % Choices(['99', '97', 'dynamic']),
+        'singletons': Bool,
+        },
+    outputs=[('taxonomy', FeatureData[Taxonomy]),
+             ('sequences', FeatureData[Sequence])],
+    input_descriptions={},
+    parameter_descriptions={
+        'version': 'UNITE version to download.',
+        'taxon_group': 'Just \'fungi\' or all \'eukaryotes\' in '
```

Comment:
can this parameter description be a little more descriptive? It is slightly vague as written

Reply:
How about this?

>Download a database with only 'fungi' or including all 'eukaryotes'.

Reply:
Perfect, yes!